testsuite/tests/ui/conftest.py

```diff
@@ -388,10 +389,11 @@ def set_callback_urls(auth0_client):
     Set callback urls for Auth0 application
     """
 
-    @backoff.on_predicate(backoff.fibo, lambda x: not x, 8, jitter=None)
     def _set_callback_urls(client_id, urls: list):
         auth0_client.clients.update(client_id, body={"callbacks": urls})
-        return auth0_client.clients.get(client_id)
+        # Due to slow Auth0 API/UI we need to use sleep here. Backoff is not viable option since the client
+        # updates URLs correctly but UI need to wait some time to correctly retrieve URLs.
```

Comment:
After trying to login via Auth0, UI shows error message but when you refresh page after couple of seconds everything works as it should. I think that is easier to use sleep instead of using backoff based on is_displayed on UI element and then refreshing page.